File: Python/Beginners/tempo_jogo.py
# Caio Cesar 12/11/23
# BEE 1046

entrada = input().split()

hora_ini = int(entrada[0])
hora_fim = int(entrada[1])

if hora_ini >= hora_fim:
    tempo = 24 - hora_ini + hora_fim
    print(f'O JOGO DUROU {tempo} HORA(S)')
else:
    tempo = hora_fim - hora_ini
    print(f'O JOGO DUROU {tempo} HORA(S)')

File: C++/Beginner/tempo_jogo.cpp
// Caio Cesar 29/11/23
// BEE 1047

#include <iostream>
using namespace std;

int main(){
    int hora_ini, hora_fim, tempo;

    cin >> hora_ini >> hora_fim;

    if(hora_ini >= hora_fim){
        tempo = 24 - hora_ini + hora_fim;
        cout << "O JOGO DUROU " << tempo << " HORA(S)" << endl;
    }else{
        tempo = hora_fim - hora_ini;
        cout << "O JOGO DUROU " << tempo << " HORA(S)" << endl;
    }

    return 0;
}